multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -36,23 +36,23 @@ import (
 )
 
 var (
-	ReasonNeverConnected   = "NeverConnected"
-	ReasonConnected        = "Connected"
-	ReasonDisconnected     = "Disconnected"
-	ReasonElectedLeader    = "ElectedLeader"
-	ReasonNotElectedLeader = "NotElectedLeader"
+	ReasonNeverConnected  = "NeverConnected"
+	ReasonConnected       = "Connected"
+	ReasonDisconnected    = "Disconnected"
+	ReasonConnectedLeader = "ConnectedLeader"
+	ReasonNotLeader       = "NotLeader"
 
 	TimerInterval     = 5 * time.Second
 	ConnectionTimeout = 3 * TimerInterval
 )
 
 type leaderStatus struct {
-	// electedLocalAsLeaderCluster indicates the member has elected the local cluster
-	// as the elected leader from which it watches resources for the MCS resource exchange
+	// connectedLocalAsLeaderCluster indicates the member has connected the local cluster
+	// as the leader from which it watches resources for the MCS resource exchange
 	// pipeline.
-	electedLocalAsLeaderCluster v1.ConditionStatus
-	message                     string
-	reason                      string
+	connectedLocalAsLeaderCluster v1.ConditionStatus
```

Comment:
Probably name it "leaderAndConnected" or just "ConnectedLeader" to be consistent with the reason.

Reply:
Renamed to `ConnectedLeader`